src/in-depth/exit-code.md

```diff
@@ -1 +1,55 @@
 # Exit codes
+
+A program doesn't always succeed.
+And when an error occurs,
+you should make sure to emit the necessary information correctly.
+In addition to
+[telling the user about errors](human-communication.md),
+on most systems,
+when a process exits,
+it also emits an exit code
+(an integer between 0 and 255).
+You should try to emit the correct code
+for your program's state.
+For example,
+in the ideal case when your program succeeds,
+it should exit with `0`.
+
+When an error occurs, it gets a bit more complicated, though.
+In the wild,
+a lot of tools exit with `1` when a general failure ocurred.
+Currently, Rust set and exit code of `101` when the process panicked.
+Beyond that, people have done many things in their programs.
+
+So, what to do?
+The BSD ecosystem has collected a common definition for their exit codes
+in a system-provided header file called [`sysexits.h`]
```

Comment:
Is it appropriate to assume the reader will understand "a system-provided header file"? A lot of people certainly will, but somebody who comes here right after finishing TRPL might not.

If this book is aimed at an audience that already knows what header files are, that's fine, I just thought I'd mention it.

Reply:
Good point! I wanted to point to the source of the exit code list but you're right that is probably more confusing than helpful.